src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php

```diff
@@ -102,4 +102,9 @@ final protected function serialize(): void
     final protected function unserialize(string $data): void
     {
     }
+
+    public function reset()
```

Comment:
```suggestion
    public function reset(): void
```

Reply:
I already tried, and I cannot :( 

```
>…regoire/dev/github.com/lyrixx/symfony(data-collector-refactor *) git di
diff --git a/src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php b/src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php
index a89d4a377c..18ac9e3850 100644
--- a/src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php
+++ b/src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php
@@ -103,7 +103,7 @@ abstract class DataCollector implements DataCollectorInterface
     {
     }
 
-    public function reset()
+    public function reset(): void
     {
         $this->data = [];
     }
>…regoire/dev/github.com/lyrixx/symfony(data-collector-refactor *) ./phpunit src/Symfony/Component/HttpKernel/
PHPUnit 9.6.10 by Sebastian Bergmann and contributors.

Testing /home/gregoire/dev/github.com/lyrixx/symfony/src/Symfony/Component/HttpKernel
.............................................................   61 / 1131 (  5%)
.............................................................  122 / 1131 ( 10%)
.............................................................  183 / 1131 ( 16%)
.............................................................  244 / 1131 ( 21%)
.............................................................  305 / 1131 ( 26%)
.....................................................PHP Fatal error:  Declaration of Symfony\Component\HttpKernel\DataCollector\RouterDataCollector::reset() must be compatible with Symfony\Component\HttpKernel\DataCollector\DataCollector::reset(): void in /home/gregoire/dev/github.com/lyrixx/symfony/src/Symfony/Component/HttpKernel/DataCollector/RouterDataCollector.php on line 54

Fatal error: Declaration of Symfony\Component\HttpKernel\DataCollector\RouterDataCollector::reset() must be compatible with Symfony\Component\HttpKernel\DataCollector\DataCollector::reset(): void in /home/gregoire/dev/github.com/lyrixx/symfony/src/Symfony/Component/HttpKernel/DataCollector/RouterDataCollector.php on line 54

Legacy deprecation notices (24)
```

Reply:
`@return void` then so that we can add it in 7.0 (don't miss updating `.github/expected*`)

Reply:
Done, thx ✅